### PR TITLE
Categories slider

### PR DESCRIPTION
**Pull request by bayan-404**
add categories slider for the home page #37  #40

### CODE REVIEW

File: client/src/Components/Common/categoriesSlider/CategoriesSlider.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import Slider from 'react-slick';
+import {Button} from '@material-ui/core'
```
Comment (someyaaltous) [inline client/src/Components/Common/categoriesSlider/CategoriesSlider.js:3]:
```
import '../../../../node_modules/slick-carousel/slick/slick.css';
import "slick-carousel/slick/slick-theme.css";
```
PROBLEM

**Comment by someyaaltous** [comment]
Name of folder, file and function doesn't match !!
```
CategoriesSlider
```

File: client/src/Components/Common/categoriesSlider/CategoriesSlider.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+import '../../../../node_modules/slick-carousel/slick/slick.css';
+import "slick-carousel/slick/slick-theme.css";
```
Comment (someyaaltous) [inline client/src/Components/Common/categoriesSlider/CategoriesSlider.js:5]:
```

import '../../../../node_modules/slick-carousel/slick/slick.css';
import "slick-carousel/slick/slick-theme.css";
```
PROBLEM

**Comment by leggomuhgreggo** [comment]
Excellent choice of [slider](https://github.com/kenwheeler/slick/commits/master?after=7daf56cb83e1365e7b7b0ef551f4a6b6b82a0f78+14) ❤️

**Pull request merged**